core/src/impl/Kokkos_QuadPrecisionMath.hpp

```diff
@@ -32,30 +30,6 @@
 #error __float128 not supported on this host
 #endif
 
-namespace Kokkos {
-template <>
-struct reduction_identity<__float128> {
-  KOKKOS_FUNCTION constexpr static __float128 sum() noexcept { return 0; }
-  KOKKOS_FUNCTION constexpr static __float128 prod() noexcept { return 1; }
-  KOKKOS_FUNCTION constexpr static __float128 max() noexcept {
-    using namespace Kokkos::Experimental;
-#if __FINITE_MATH_ONLY__
-    return finite_min_v<__float128>;
-#else
-    return -infinity_v<__float128>;
-#endif
-  }
-  KOKKOS_FUNCTION constexpr static __float128 min() noexcept {
-    using namespace Kokkos::Experimental;
-#if __FINITE_MATH_ONLY__
-    return finite_max_v<__float128>;
-#else
-    return infinity_v<__float128>;
-#endif
-  }
-};
-}  // namespace Kokkos
```

Comment:
The pull request description didn't make it clear that dropping `KOKKOS_FUNCTION` was a deliberate choice.

Reply:
Good point.  I actually overlooked we had these function annotations.
I think that was not intentional and I don't think there was any test coverage for it.
I will update the description.